src/particles/deposition/PlasmaDepositCurrentInner.H

```diff
@@ -97,6 +104,7 @@ void doDepositionShapeN (const PlasmaParticleIterator& pti,
     const amrex::Real dyi = 1.0/dx[1];
     const amrex::Real dzi = 1.0/dx[2];
     const amrex::Real invvol = Hipace::m_normalized_units ? 1. : dxi*dyi*dzi;
+    const amrex::GpuArray<amrex::Real, 3> dx_arr = {dx[0], dx[1], dx[2]};
```

Comment:
Let's avoid storing more temporary variables than necessary, these do not come for free. Could you reduce the redundancy here? By defining `dxi, dyy, dzi, invvol, dx_arr` the compiler may store 7 `Real`s where only 3 are needed. If you prefer, you can just keep it that way for simplicity, but just open an issue on that.